xmlapi/cgi.tcl

```diff
@@ -1335,30 +1335,30 @@ proc cgi_input {{fakeinput {}} {fakecookie {}}} {
 	    set _cgi_cookie($varname) $val
 	}
     }
-}
-
-set _cgi_read_line_buffer ""
-proc _cgi_read_line {fin bufvar crlfvar} {
-    global _cgi_read_line_buffer
-    upvar $bufvar buffer
-    upvar $crlfvar crlf
-    set line_end [string first "\r\n" $_cgi_read_line_buffer]
-    while {($line_end < 0 ) && ![eof $fin]} {
-        append _cgi_read_line_buffer [read $fin 65536]
-        set line_end [string first "\r\n" $_cgi_read_line_buffer]
-    }
-    if {$line_end >= 0} { 
-        incr line_end -1
-        set buffer [string range $_cgi_read_line_buffer 0 $line_end]
-        set crlf "\r\n"
-        set _cgi_read_line_buffer [string range $_cgi_read_line_buffer [expr $line_end + 3] end]
-        return 1
-    } else {
-        set buffer $_cgi_read_line_buffer
-        set crlf ""
-        set _cgi_read_line_buffer ""
-        return [expr ([string length $buffer] > 0)]
-    }
+}
+
+set _cgi_read_line_buffer ""
```

Comment:
Exactly. Just converting Windows line feeds (CRLF) to Unix/Linux line feeds (LF).